packages/calling/src/CallingClient/calling/callManager.ts

```diff
@@ -260,10 +266,12 @@ export class CallManager extends Eventing<CallEventTypes> implements ICallManage
           } else {
             /* If Call.Media arrived before Call.Setup , we create the Call Object here */
 
+            const line = this.getLine(mobiusEvent.data.deviceId);
```

Comment:
we can directly fetch lineId here instead of line object

Reply:
Changed the function getLine to return only lineId